tests/bell.py

```diff
@@ -119,6 +119,7 @@ def add_member(driver, first_name = "", last_name = "", middle_name = "",
     the member already exists, return a tuple of empty strings.
     TODO: Needs testing AND refactoring!
     """
+    birthdate = birthdate or []
```

Comment:
This line is a bug ;-)
The birthdate variable will never take up the value of `[]`, thus the conditional below `if birthdate == []:` will never be true.
Rather, delete this line and change the conditional below to `if birthdate == None:`